javascript/chronometer.js

```diff
@@ -1,33 +1,51 @@
 class Chronometer {
   constructor() {
+    this.currentTime = 0;
+    this.intervalId = null;
     // ... your code goes here
   }
 
   start(callback) {
```

Comment:
You missed to dd the callback logic. It is said that the callback is a function that can be optionally passed as an argument. So the idea was to check if there is a callback passed, if yes it should be executed inside the setInterval. Like this:

```
  start(callback) {
    this.intervalId = setInterval(() => {
      this.currentTime++;
      if (callback) {
        callback();
      }
    }, 1000);
  }

```